src/renderer/components/NodeSelectorPanel/RepresentativeNode.tsx

```diff
@@ -40,8 +43,26 @@ export const RepresentativeNode = memo(
             bgGradient = `linear-gradient(90deg, ${accentColor} 0%, ${bgColor} 100%)`;
         }
 
+        const ref = useRef(null);
+
+        useHotkeys(
+            'enter',
+            () => {
+                if (document.activeElement === ref.current) {
+                    createNodeFromSelector();
+                }
+            },
+            [ref]
+        );
+
         return (
             <Center
+                _active={{
+                    outlineColor: accentColor,
+                }}
+                _focus={{
+                    outlineColor: accentColor,
+                }}
                 _hover={{
                     outlineColor: accentColor,
                 }}
```

Comment:
Could we make these a single line each?

```suggestion
                _active={{ outlineColor: accentColor }}
                _focus={{ outlineColor: accentColor }}
                _hover={{ outlineColor: accentColor }}
```